flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/MapTypeTest.scala

```diff
@@ -139,13 +139,6 @@ class MapTypeTest extends MapTypeTestBase {
   @Test
   def testMapOperations(): Unit = {
 
-    // comparison
-    testAllApis(
```

Comment:
Why did this test ever worked?

Reply:
That was my bad, I was only testing the invalid equality case ( e.g. `MAP[STRING, INT] = MAP[STRING,STRING]` should throw ValidationException), I did not put in the valid equality test :-(